src/Tribe/Plugins.php

```diff
@@ -131,5 +131,136 @@ public function get_list() {
 			return apply_filters( 'tribe_plugins_get_list', $this->tribe_plugins );
 		}
 
+		/**
+		 * Returns true if the plugin "Event Tickets" is active.
```

Comment:
```suggestion
		 * Returns true if the plugin "Event Tickets" is active.
		 *
```

Blank line between short and long/detail description.

Reply:
Methods will be redone